### PR TITLE
Create d/README.source

### PR DESCRIPTION
**Pull request by smoe**
This explains
 * that the focus of your community and of your debian packaging remains on github
 * that you invite whoever is interested to join you there
 
and presents a recipe how to prepare the package that is ready to upload from what the repository presents.

### CODE REVIEW

File: debian/README.source [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   cd $WORKDIR
+
+2. Unpack
+   tar xJvf mesaflash_*.orig.tar.xz
+
+3. Slightly adapt for unstable
+   bash
+   cd $WORKDIR/mesaflash
+   rm debian/compat
+   sed -i 's/debhelper .*,/debhelper-compat (= 13)/' debian/control
```
Comment (SebKuzminsky) [inline debian/README.source:33]: Should be `sed -i 's/debhelper .*,/debhelper-compat (= 13)/,' debian/control`, I think?  (Append a `,` to the end of the new line.)

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]
Thanks for this documentation @smoe!

In your opinion does this remove the need for your earlier commit that removes `debian/compat` and bumps debhelper from `>= 9` to `>= 10`?  @jepler  and I have been talking about the best way for us to support the old distros while also moving forward with the upload to Debian unstable, this seems simpler than what we had been planning.

**Comment by SebKuzminsky** [comment]
If you add
```
diff --git a/debian/README.source b/debian/README.source
index 547deb9..b854e6c 100644
--- a/debian/README.source
+++ b/debian/README.source
@@ -31,6 +31,7 @@ Steps
    cd $WORKDIR/mesaflash
    rm debian/compat
    sed -i 's/debhelper .*,/debhelper-compat (= 13),/' debian/control
+   sed -i 's/^Standards-Version: .*$/Standards-Version: 4.6.0/' debian/control
    sed -i '1s/ stable;/ unstable;/' debian/changelog
```
and we restore the Standards-Version to the embarrassingly old version we had before, then the master branch will keep building on our old distros and the modified version resulting from the "manual script" in the README will be valid for unstable, I think?

**Comment by smoe** [comment]
> If you add
> 
> ```
> diff --git a/debian/README.source b/debian/README.source
> index 547deb9..b854e6c 100644
> --- a/debian/README.source
> +++ b/debian/README.source
> @@ -31,6 +31,7 @@ Steps
>     cd $WORKDIR/mesaflash
>     rm debian/compat
>     sed -i 's/debhelper .*,/debhelper-compat (= 13),/' debian/control
> +   sed -i 's/^Standards-Version: .*$/Standards-Version: 4.6.0/' debian/control
>     sed -i '1s/ stable;/ unstable;/' debian/changelog
> ```
> 
> and we restore the Standards-Version to the embarrassingly old version we had before, then the master branch will keep building on our old distros and the modified version resulting from the "manual script" in the README will be valid for unstable, I think?

Nice catch. Yes. Suggestion:
```
sed -i '/^Standards-Version:/s/ [0-9.]*/ 4.6.0/' debian/control
```

File: debian/README.source (hunk 1 above)
Comment (smoe) [reply debian/README.source]: fixed

**Comment by SebKuzminsky** [comment]
Thanks @smoe!

**Pull request merged**